rollup.config.js

```diff
@@ -6,46 +6,63 @@ const { version } = require('./package.json');
 const terserConfig = {
 	compress: true,
 	mangle: true
-}
+};
 
 function bundle(type) {
 	let minify = false;
 	let format = type.replace(".min", () => {
 		minify = true;
-		return ""
+		return "";
 	});
 
 	let suffix = `.${type}`.replace(".esm", "");
+	let folder = format === 'esm' ? '' : `${format}/`;
 
 	return {
-		file: `dist/jsep${suffix}.js`,
+		file: `dist/${folder}/jsep${suffix}.js`,
```

Comment:
Note that if `folder` is empty, this would include a double slash (`//`). Which I guess works fine, but just wanted to point it out.

Reply:
I was thinking of that when I wrote that, but then forgot to update it! Fixed now :-)